docs/guides/aws-workspace.md

```diff
@@ -258,7 +267,28 @@ As a workaround give the `aws_iam_role` more time to be created with a `time_sle
 
 ```hcl
 resource "time_sleep" "wait" {
-  depends_on = [aws_iam_role.cross_account_role]
+  depends_on = [
+    aws_iam_role.cross_account_role]
   create_duration = "10s"
+}
+```
+
+#### IAM policy error
+
+If you notice below error:
+
+```
+Error: MALFORMED_REQUEST: Failed credentials validation checks: Spot Cancellation, Create Placement Group, Delete Tags, Describe Availability Zones, Describe instances, Describe Instance Status, Describe Placement Group, Describe Route Tables, Describe Security Groups, Describe Spot Instances, Describe Spot Price History, Describe Subnets, Describe Volumes, Describe Vpcs, Request Spot Instances
 ```
 
+- Try creating workspace from UI:
+
+<p align="center">
+  <img src="https://github.com/databrickslabs/terraform-provider-databricks/raw/master/docs/images/create_workspace_error.png">
```

Comment:
i'm not sure if we need this image here - we're trying to fully automate deployment?..

Reply:
This was just for troubleshooting. In case users are trying to understand the issue.